script/semantic-conventions/templates/trace_deprecations.php.partial

```diff
@@ -3,31 +3,11 @@
      */
     public const FAAS_EXECUTION = 'faas.execution';
 
-    /**
```

Comment:
all these guys were re-added to semconv as deprecated, so no longer needed here.